cmd/createcluster.go

```diff
@@ -91,6 +94,7 @@ func newCreateClusterCmd(runFunc func(context.Context, io.Writer, clusterConfig)
 func bindClusterFlags(flags *pflag.FlagSet, config *clusterConfig) {
 	flags.StringVar(&config.Name, "name", "", "The cluster name")
 	flags.StringVar(&config.ClusterDir, "cluster-dir", ".charon/cluster", "The target folder to create the cluster in.")
+	flags.StringVar(&config.DefFile, "definition-file", "", "The path to the file or an HTTP URL to fetch cluster definition.")
```

Comment:
this is not a definition file, it is a partial definition, suggest not supporting local files yet (since they should never actually exist). Also not sure we want to leak the term "partial definition" to users, since it isn't really a well thought out thing. 

Suggest `--config-url: Optional HTTP URL to fetch configuration overriding all other configuration flags.`